### PR TITLE
acknowledgements

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
#268

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,6 +24,21 @@
...
+        name: "Bogdan Drutu",
+        company: "Google",
+        companyURL: "https://google.com"
+        //TODO: w3cid is missing
+      },
+      {
+        name: "Nik Molnar",
+        company: "Microsoft",
+        companyURL: "https://microsoft.com"
+        //TODO: w3cid is missing
```
Comment (nikmd23) [inline index.html]:
```suggestion
        w3cid: "104238"
```

**Review by nikmd23 — APPROVED** [review]
Looks good to me. I've added in my `w3cid`.

**Review by AloisReitbauer — APPROVED** [review]

**Review by danielkhan — APPROVED** [review]
LGTM

**Review by watson — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
@nikmd23 can you find one for Bogdan? How did you find it?

File: index.html [hunk 2]
```diff
@@ -24,6 +24,21 @@
         company: "Dynatrace",
         companyURL: "https://dynatrace.com",
         w3cid: "48276"
+      },
+      {
+        name: "Bogdan Drutu",
+        company: "Google",
+        companyURL: "https://google.com"
+        //TODO: w3cid is missing
```
Comment (SergeyKanzhelev) [inline index.html]: @plehegar please help here

File: index.html [hunk 3]
```diff
@@ -24,6 +24,21 @@
...
+      },
+      {
+        name: "Nik Molnar",
+        company: "Microsoft",
+        companyURL: "https://microsoft.com"
+        w3cid: "104238"
+      },
+      {
+        name: "Yuri Shkuro" //TODO: as invited expert
```
Comment (SergeyKanzhelev) [inline index.html]: @plehegar please help here

File: index.html (hunk 2 above)
Comment (bogdandrutu) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (bogdandrutu) [reply index.html]: 1. Go to https://www.w3.org/Member/ - use your credentials to login
1. Then on the right side press User Account
1. A personal page will open, look in the URL `https://www.w3.org/users/XXXXXX` where `XXXXXX` will be your w3cid

**Review by bogdandrutu — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (SergeyKanzhelev) [reply index.html]: Cc: @yurishkuro

Comment (yurishkuro) [reply index.html]: My id is https://www.w3.org/users/104074

**Review by yurishkuro — APPROVED** [review]

**Review by discostu105 — APPROVED** [review]

**Pull request merged**